examples/README.md

```diff
@@ -1,4 +1,68 @@
-# Doorman Integration Examples
+# Integration Examples
 
 - [Python / Flask](python/): A Web UI interacts with Auth0 and a Flask API
 
+# Configuration Examples
+
+## Superusers with Doorman tags
+
+We define a tag ``superuser`` and use it in principals in policies rules.
+
+```yaml
+
+    service: https://api.service.org
+    tags:
+      superuser:
+        - userid:maria
+        - group:admins
+    policies:
+      -
+        id: super-users
+        description: Superusers can do everything
+        principals:
+          - tag:superuser
+        actions:
+          - <.*>
+        resources:
+          - <.*>
+```
+
+## Superusers from service
+
+The superusers are managed on the service itself (eg. database). We will rely on a `superuser` role. The service can send user roles in authorization requests context.
```

Comment:
I don't understand these sentences. In particular "We will rely on a `superuser` role."
Does it mean when the service maintains its own database of user groups/roles/principles and it includes these when it queries Doorman? (by the way, if that's true, what's the point of asking Doorman?)

Perhaps I'm not versed in Pyramid lingo but I find it confusing to juggle the term "role" *and* the term "principle".

Reply:
I'll rework that part.

A [principal](https://en.wikipedia.org/wiki/Principal_(computer_security)) is a generic term that generalizes userid, groups, tags, roles.

A role is a principal that usually depends on the relation between a user and a resource. For example, `alice` is the `role:reporter` of `project/42/issues/2`. It's not the responsability of the identity provider nor Doorman to manage this relation. The service sends the roles as «context».